src/helpers/bem-helper.js

```diff
@@ -6,3 +9,17 @@ export function BEMClass(componentName) {
     return className;
   };
 }
+
+export function createBemBlockHelper(block, { isConsume } = {}) {
```

Comment:
I think it will be better not to add this helper right now because we moving to css modules

Reply:
For a while it moved locally to slider.
I'll remove it during switch from BEM to CSS Modules (see one of previous comments)

_Issue can be resolved._